internal/cli/resume.go

```diff
@@ -0,0 +1,195 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	api "github.com/flightctl/flightctl/api/v1alpha1"
+	apiclient "github.com/flightctl/flightctl/internal/api/client"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+type ResumeOptions struct {
+	GlobalOptions
+	LabelSelector string
+}
+
+func DefaultResumeOptions() *ResumeOptions {
+	return &ResumeOptions{
+		GlobalOptions: DefaultGlobalOptions(),
+		LabelSelector: "",
+	}
+}
+
+func NewCmdResume() *cobra.Command {
+	o := DefaultResumeOptions()
+	cmd := &cobra.Command{
+		Use:   "resume (device/NAME | devices)",
+		Short: "Resume a device or devices based on label selector.",
+		Long: `Resume devices that are in a conflictPaused state.
+
+Examples:
+  # Resume a specific device
+  flightctl resume device/my-device
+
+  # Resume all devices with a specific label
+  flightctl resume devices --selector env=production
+
+  # Resume devices with multiple label conditions
+  flightctl resume devices --selector env=staging,tier!=api`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := o.Complete(cmd, args); err != nil {
+				return err
+			}
+			if err := o.Validate(args); err != nil {
+				return err
+			}
+			ctx, cancel := o.WithTimeout(cmd.Context())
+			defer cancel()
+			return o.Run(ctx, args)
+		},
+		SilenceUsage: true,
+	}
+	o.Bind(cmd.Flags())
+	return cmd
+}
+
+func (o *ResumeOptions) Bind(fs *pflag.FlagSet) {
+	o.GlobalOptions.Bind(fs)
+	fs.StringVarP(&o.LabelSelector, "selector", "l", o.LabelSelector, "Selector (label query) to filter on, supporting operators like '=', '!=', and 'in' (e.g., -l='key1=value1,key2!=value2,key3 in (value3, value4)').")
+}
+
+func (o *ResumeOptions) Complete(cmd *cobra.Command, args []string) error {
+	if err := o.GlobalOptions.Complete(cmd, args); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (o *ResumeOptions) Validate(args []string) error {
+	if err := o.GlobalOptions.Validate(args); err != nil {
+		return err
+	}
+
+	arg := args[0]
+
+	// Handle bulk resume case
+	if arg == "devices" {
+		if o.LabelSelector == "" {
```

Comment:
FieldSelector is also valid.
But is there any reason to enforce this?  Maybe they just want to unpause all devices?